ocaml/idl/datamodel_repository.ml

```diff
@@ -94,6 +95,27 @@ let introduce_bundle =
     ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
     ()
 
+let introduce_remote_pool =
+  call ~name:"introduce_remote_pool" ~in_oss_since:None ~lifecycle:[]
+    ~doc:"Add the configuration for a new remote pool repository"
+    ~params:
+      [
+        (String, "name_label", "The name of the repository")
+      ; (String, "name_description", "The description of the repository")
+      ; ( String
+        , "address"
+        , "The hostname of the coordinator of the remote pool repository"
```

Comment:
Might want to include an example for the format. Is this an IP address? A URL? A fqn?

Reply:
The "address" here will be the same type as the "master_address" of `pool.join`, and the description was copied from there.
Looked into the code, I think it will support both "host name" and "IP address", the description is updated.